framework/caching/DbCache.php

```diff
@@ -286,4 +294,36 @@ protected function flushValues()
 
         return true;
     }
+
+    /**
+     * @return bool
+     *
+     * Checking Mssql: if field is varbinary - return true
+     */
+    protected function isVarbinaryDataField()
+    {
+        return (
+            in_array($this->db->getDriverName(), ['sqlsrv', 'dblib']) &&
+            $this->db->getTableSchema('cache')->columns['data']->dbType === 'varbinary');
```

Comment:
```suggestion
        return in_array($this->db->getDriverName(), ['sqlsrv', 'dblib'])
            && $this->db->getTableSchema('cache')->columns['data']->dbType === 'varbinary';
```

Reply:
Also please store the result in a private property. This method is used several times here and while some parts of the conditions are inner-cached already there are the ones that are recalculated every time.

Reply:
Resolved